skimage/future/graph/rag.py

```diff
@@ -264,6 +272,30 @@ def _add_node_silent(self, n):
         .. seealso:: :func:`networkx.Graph.add_node`."""
         super(RAG, self).add_node(n)
 
+    def nodes_iter(self, *args, **kwargs):
```

Comment:
I'm not convinced this is needed...? In general, I don't want to try to maintain an interface to multiple versions of networkx, but rather for RAG to mirror the interface of whatever version the user has installed. What's your rationale here, @matthew-brett?